### DISCUSSION TITLE
Raspberry Pi Zero 2 W

### DISCUSSION THREAD

**Discussion by EdNekebno** [EdNekebno/DGTCentaurMods#74] [General]
They just launched a Raspberry Pi Zero 2 W! The Raspberry Pi Zero W is still going to be sold. Ironically, at the moment Zero W's are nearly impossible to find but an online electronics place for this country did have stock of Pi Zero 2 W's (at a limit of one per customer!). So I ordered one...should come before the weekend so I might stand a chance of testing it over the weekend.

The claim is it is 5x faster. I'm not sure that that has a big impact on this project - eink is always going to be the limiting factor. It could affect UCI engine play though or what engines we can run, as the processing power is better. But mostly I just want to test if it works as it is what people will most likely have available to them.

**Comment by wormstein** [comment]
Just ordered myself one too. I guess you talk about kiwi electronics. They rock!

More power is good for compiling SF onboard and helps chess engines.

As long the OS is same i don't see problems with the package deployment.

**Reply by EdNekebno** [reply]
Yeh....I hadn't consider SF compile speeds. That could definitely do with being 5 times faster!
And yup...kiwi electronics. There's a message on their site now that says because of the busyness from the release they're not able to send all of the orders the same day. Fingers crossed. It's strange how the moment I order something I want it to arrive desperately even though I have no real need of it straight away :)

**Comment by EdNekebno** [comment]
Just thought I'd say - it works. I needed to make some small changes in the menu.py timings. It turns out I missed a thought here. Although the Pi Zero 2 W's extra speed doesn't have much of an effect on the software. There's two key aspects where it does seem to have an effect:

1. The wifi/bluetooth seems better.
2. Boot speed. And this I forgot about. The start up time for the system is much quicker. 

Then my opinion is that the extra money for a Pi Zero 2 W over a Pi Zero W is more than worth it.

**Reply by wormstein** [reply]
Sounds great. Mine it's on its way! Yay!

**Comment by EdNekebno** [comment]
I get a 28 second boot time by the way: https://twitter.com/ednekebno/status/1456568863882158087
